sda/conode_test.go

```diff
@@ -0,0 +1,30 @@
+package sda
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestConode_ProtocolRegisterName(t *testing.T) {
```

Comment:
`conode.go` and `protocol.go` are 100% covered - I feel like I should get a gold medal ;)